data/spelling-dict/hunspell/pt_PT_45.aff

```diff
@@ -1270,7 +1270,283 @@ SFX K   0               mos             dizer           +P=1,N=p,T=ip
 SFX K   0               des             dizer           +P=2,N=p,T=ip
 SFX K   0               em              dizer           +P=3,N=p,T=ip
 
-SFX L Y 77
+SFX L Y 353
+SFX L   er      �-la-�         ter
+SFX L   er      �-las-�        ter
+SFX L   0       -lhe-�         ter
+SFX L   er      �-lo-�         ter
+SFX L   er      �-los-�        ter
+SFX L   0       -me-�          ter
+SFX L   0       -nos-�         ter
+SFX L   0       -te-�          ter
+SFX L   0       -vos-�         ter
+SFX L   0       -lhes-�o       ter
+SFX L   0       -me-�o         ter
+SFX L   0       -nas-�o        ter
+SFX L   0       -nos-�o        ter
+SFX L   0       -se-�o         ter
+SFX L   0       -te-�o         ter
+SFX L   0       -vos-�o        ter
+SFX L   0       -lhes-ei       ter
+SFX L   0       -me-ei         ter
+SFX L   0       -te-ei         ter
+SFX L   0       em-lhes        ter
+SFX L   0       em-no          ter
+SFX L   0       em-nos         ter
+SFX L   0       em-te          ter
+SFX L   er      �-la-ei        ter
+SFX L   er      �-las-ei       ter
+SFX L   er      �-lo-ei        ter
+SFX L   er      �-los-ei       ter
+SFX L   er      �-la-ia        ter
+SFX L   er      �-las-ia       ter
+SFX L   er      �-lo-ia        ter
+SFX L   er      �-los-ia       ter
+SFX L   er      �-la-iam       ter
+SFX L   er      �-las-iam      ter
+SFX L   er      �-lo-iam       ter
+SFX L   er      �-los-iam      ter
+SFX L   er      �-la-emos      ter
+SFX L   er      �-lo-emos      ter
+SFX L   er      �-los-emos     ter
+SFX L   er      �-las-emos     ter
+SFX L   er      �-lo-�amos     ter
+SFX L   er      �-la-�amos     ter
+SFX L   er      �-los-�amos    ter
+SFX L   er      �-las-�amos    ter
+SFX L   0       -lhe-emos      ter
+SFX L   0       -lhes-emos     ter
+SFX L   0       -nos-emos      ter
+SFX L   0       -te-emos       ter
+SFX L   0       -lhes-ia       ter
+SFX L   0       -me-ia         ter
+SFX L   0       -nos-ia        ter
+SFX L   0       -se-ia         ter
+SFX L   0       -vos-ia        ter
+SFX L   0       -lhe-iam       ter
+SFX L   0       -me-iam        ter
+SFX L   0       -se-iam        ter
+SFX L   0       -te-iam        ter
+SFX L   0       -nos-�amos     ter
+SFX L   0       mos-lhe        ter
+SFX L   0       mos-lhes       ter
+SFX L   0       mos-te         ter
+SFX L   er      �m-lhe         ter
+SFX L   er      �m-lhes        ter
+SFX L   er      �m-me          ter
+SFX L   er      �m-na          ter
+SFX L   er      �m-nas         ter
+SFX L   er      �m-no          ter
+SFX L   er      �m-nos         ter
+SFX L   er      �m-se          ter
+SFX L   er      �m-te          ter
+SFX L   er      �m-vos         ter
```

Comment:
This outputs `tém-se`, etc. These should only be available for derived forms of `ter`, but not `ter` itself.